Tests/Data/geometry-hierarchy-map.json

```diff
@@ -12,24 +12,24 @@
             }
         },
         {
-            "volume": 1,
-            "layer": 2,
+            "Volume": 1,
```

Comment:
I liked the original "volume" , "layer" better

Reply:
Me too but the issue is that when you write the json file the key are ordered alphabetically which mean it use to look like : 
```
            "layer": 4,
            "sensitive": 5,
            "value": {
                "x": 3.25,
                "y": -5
            }
            "volume": 3,
``` 
Which was confusing. I don't know if there is a way to force the writing to not order the key.

Reply:
this now fixed with PR #686